pkgs/development/python-modules/find-libpython/default.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "find_libpython";
+  version = "0.3.0";
+  format = "setuptools";
```

We should stick to supported Python releases. The change is high that the module will not be used/tests on old Python releases.

```suggestion
  format = "setuptools";

  disabled = pythonOlder "3.7";
```